libs/core/src/lib/card/card-loader.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, ChangeDetectionStrategy, ElementRef, Renderer2 } from '@angular/core';
+
+import { CLASS_NAME } from './constants';
+
+@Component({
+    selector: 'fd-card-loader',
+    template: '<ng-content></ng-content>',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class CardLoaderComponent implements OnInit {
+    /** @hidden */
+    constructor(private _elementRef: ElementRef<HTMLElement>, private _renderer: Renderer2) {}
+
+    /** @hidden */
+    ngOnInit(): void {
+        this._addClassNameToHostElement(CLASS_NAME.cardLoader);
+    }
+
+    /**@hidden */
+    private _addClassNameToHostElement(className: string): void {
+        this._renderer.addClass(this._elementRef.nativeElement, className);
```

Comment:
should use cssClassBuilder (same throughout this pr)

Reply:
If it's only one class you can use event `@HostBinding()`, but I would like to hear the reason why renderer is used there.

Reply:
moved to `CssClassBuilder`